py-polars/polars/utils/various.py

```diff
@@ -392,7 +392,7 @@ def __get__(  # type: ignore[override]
             return self.fget(  # type: ignore[misc]
                 instance if isinstance(instance, cls) else cls
             )
-        except AttributeError:
+        except (AttributeError, ImportError):
```

Comment:
adding this to avoid an importerror if building the docs without `hvplot` installed. I don't think `hvplot` should be required for building docs, and it's probably not worth showing plots in the docs themselves